src/NumberInput/NumberInput.tsx

```diff
@@ -158,7 +160,12 @@ export const NumberInput: FC<NumberInputProps> = ({
   }
 
   return (
-    <Container className={className} hasLabel={!!label} hasError={!!errorMsg}>
+    <Container
```

Comment:
this doesn't seem to be standardised with the new Field?

Reply:
Yeah, I think it should be migrated but didn't want to do it on this PR since it was getting quite big already